src/other_functions.jl

```diff
@@ -178,3 +179,70 @@ polynomials. For more details, see [`Base.isapprox`](@ref).
 function isapprox{T<:AbstractSeries,S<:AbstractSeries}(x::T, y::S; rtol::Real=rtoldefault(x,y), atol::Real=0, nans::Bool=false)
     x == y || (isfinite(x) && isfinite(y) && norm(x-y,1) <= atol + rtol*max(norm(x,1), norm(y,1))) || (nans && isnan(x) && isnan(y))
 end
+
+
+#taylor_expand function for Taylor1
+doc"""
+    taylor_expand(f,x0;order)
+
+Makes a Taylor expansion of the function `f` around the point `x0`. If x0 is a scalar,
+a `Taylor1` expansion will be done. If `x0` is a vector, a `TaylorN` expansion will be
+computed, changing the number of `TaylorN` variables
+according to the dimension of `f`.
+"""
+function taylor_expand(f::Function; order::Int64=15)
+   a = Taylor1(order)
+   return f(a)
+end
+
+function taylor_expand{T<:Number}(f::Function, x0::T; order::Int64=15)
+   a = Taylor1([x0, one(T)], order)
+   return f(a)
+end
+
+#taylor_expand function for TaylorN
+function taylor_expand{T<:Number}(f::Function, x0::Vector{T}; order::Int64=get_order()) #a Taylor expansion around x0
+    ll = length(x0)
+    ll == get_numvars() ? X = get_variables() : begin
+        X = set_variables("x",order=order,numvars=ll)
+        warn("Changed number of TaylorN variables to $ll.")
+        end
+    return f(X.+x0)
+end
+
+function taylor_expand(f::Function, x0...; order::Int64=get_order()) #a Taylor expansion around x0
+    ll = length(x0)
+    ll == get_numvars() ? X = get_variables() : begin
+        X = set_variables("x",order=order,numvars=ll)
+        warn("Changed number of TaylorN variables to $ll.")
+        end
+
+    return f(x0 .+ X...)
+end
+
+#taylor_expand! function for Taylor1
+doc"""
+    taylor_expand!(a,x0;order)
+
+Takes `a <: Union{Taylo1,TaylorN}` and expands it around the coordinate `x0`.
+"""
+function taylor_expand!(a::Taylor1)
```

Comment:
What about naming this function `update!` (as proposed in #80), and export it? Since this method is some sort of fallback, i think it can be written so `a::Union{Taylor1, TaylorN}`, which also covers another method below.